doc/ref/helpintf.xml

```diff
@@ -217,6 +217,11 @@ gap> stream := InputTextFile( file );;
   <A>type</A> which currently must be handled and the corresponding result format
   are described in the list below.
 </Item>
+<Mark><C>SubsectionNumber( <A>info</A>, <A>i</A> )</C> </Mark>
+<Item>
+  This returns some &GAP; object that identifies the position in the book
+  where the display of this entry is started.
```

Comment:
While factually correct, this is not very helpful for somebody who wants to implement a help book handler. In particular,  it does not say what the result of `SubsectionNumber` is used for, and what it must be capable of. So perhaps add: "The help system may coalesce multiple help entries for which `SubsectionNumber` returns the same object."